enrichments/__init__.py

```diff
@@ -0,0 +1,4 @@
+# Raise this to stop an alert from firing from an enrichment
```

Comment:
For clarity and readability sake, can you make this comment the actual docstring for the `class` (and drop the `pass`), like:
```
class DropAlertException(Exception):
    """Raise this to stop an alert from firing from an enrichment"""
```